src/target.zig

```diff
@@ -310,6 +310,31 @@ pub fn defaultCompilerRtOptimizeMode(target: std.Target) std.builtin.OptimizeMod
     }
 }
 
+pub fn canBuildLibCompilerRt(target: std.Target, use_llvm: bool, have_llvm: bool) bool {
+    switch (target.os.tag) {
+        .plan9 => return false,
+        else => {},
+    }
+    switch (target.cpu.arch) {
+        .spirv, .spirv32, .spirv64 => return false,
+        .amdgcn => return false,
+        else => {},
+    }
```

Comment:
```suggestion
    switch (target.cpu.arch) {
        .spirv, .spirv32, .spirv64 => return false,
        .amdgcn => return false,
        .nvptx, .nvptx64 => return false,
        else => {},
    }
```

Or was this intentional?

Reply:
Intentional. *Compiler RT* builds fine for ptx

Reply:
Ok, might want to clarify the commit message then, since it sounds like it means to disable both libraries for both targets.